library/src/main/scala/scala/scalajs/runtime/LinkingInfo.scala

```diff
@@ -10,6 +10,9 @@ trait LinkingInfo extends js.Object {
 
   /** Whether we are assuming ECMAScript 6 support or not. */
   val assumingES6: Boolean = js.native
+
+  /** Version of the linker */
+  val version: String = js.native
```

Comment:
Consider renaming to `linkerVersion`. `version` is a bit too general, even within `LinkingInfo`.